bin/servicerouter.py

```diff
@@ -27,85 +29,87 @@
 Usage:
   $ servicerouter.py --marathon http://marathon1:8080 --haproxy-config /etc/haproxy/haproxy.cfg
 
-  The user servicerouter is executed as must have the right to reload haproxy.
+  The user that executes servicerouter must have the permission to reload
+  haproxy.
 
 
 Environment Variables:
   HAPROXY_GROUP
-  Only servicerouter instances which are members of the given group will
-  point to the service. Service routers with the group '*' will collect
-      all groups.
+    The group of servicerouter instances that point to the service.
+    Service routers with the group '*' will collect all groups.
 
   HAPROXY_{n}_VHOST
-  HTTP Virtual Host proxy hostname to catch
-  Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
+    The Marathon HTTP Virtual Host proxy hostname to gather.
+    Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
 
   HAPROXY_{n}_STICKY
-  Use sticky request routing for the service
-  Ex: HAPROXY_0_STICKY = true
+    Enable sticky request routing for the service.
```

Comment:
I would like a reference to the HAPROXY configuration that results in. Is it done via cookies, IP, connection? How sticky is it ;)